02-documentation/developer-guides/building-a-nextjs-blog-with-squidex.md

```diff
@@ -0,0 +1,585 @@
+---
+description: >-
+  How to build a Next.js blog powered by content from Squidex CMS.
+---
+
+# Building a blog with Squidex and Next.js
+
+### Introduction
```

Comment:
You make a lot of introductions, which are great for a general article, but make no sense for the docs, I think.

Reply:
That's true. I sent in a commit with a shorter introduction.